woocommerce/payment-gateway/class-sv-wc-payment-gateway.php

```diff
@@ -4152,6 +4152,26 @@ public function get_checkout_order_received_order_id() {
 	}
 
 
+	/**
+	 * Returns the error message for display if the gateway is not configured.
+	 *
+	 * @since 5.4.2-dev.1
+	 *
+	 * @return string
+	 */
+	public function get_not_configured_error_message() {
+
+		return sprintf(
+			/* translators: %1$s - gateway name, %2$s - <a> tag, %3$s - </a> tag, %4$s - <a> tag */
+			__( 'Heads up! %1$s is not fully configured and cannot accept payments. Please %2$sreview the documentation%3$s and configure the %4$sgateway settings%3$s.', 'woocommerce-plugin-framework' ),
```

Comment:
I know this is technically correct, however since translations may be handled by non-devs I believe that using the same placeholder multiple times can be error prone - %3$s in this case, containing the closing </a> tag. In other occasions I see we just introduce another placeholder, so each pair of opening/closing tags have their own </a> represented by a different placeholder.